packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -213,7 +214,20 @@ module.exports = {
     // to restart the development server for Webpack to discover it. This plugin
     // makes the discovery automatic so you don't have to restart.
     // See https://github.com/facebookincubator/create-react-app/issues/186
-    new WatchMissingNodeModulesPlugin(paths.appNodeModules)
+    new WatchMissingNodeModulesPlugin(paths.appNodeModules),
+    // Trigger some typechecking if a file matches with an @ flow comment
+    new FlowTypecheckPlugin({
+      flowconfig: [
+        '[ignore]',
+        '<PROJECT_ROOT>/node_modules/fbjs/.*',
+        '',
+        '[libs]',
+        './flow-typed'
```

Comment:
>However, modifying that file doesn't seem to trigger a rebuild.

Likely because it's not part of webpack flow (it's a test after all). So it looks like this integration is a bit limited by not working with test files. Not sure if it's easily fixable or not.